test/e2e/connector.test.js

```diff
@@ -37,14 +40,26 @@ describe('Amio API Connector', function () {
     })
   })
 
+  describe('notifications', () => {
+    it('send a notification', async () => {
+
+      const type = 'typing_on'
```

Comment:
what about to test all notification types?

Reply:
I am just testing that the request for a notification passes. All notifTypes are tested in amio-communicator.